### PR TITLE
Sponser link, diary image added

### PR DESCRIPTION
**Pull request by imvinojanv**
Sponsors link added 
Lecturer board hided
Diary image added

### CODE REVIEW

**Comment by imvinojanv** [comment]
I have added the diary-3 page which wrote by Adv.Hirushani

File: contact.html [hunk 1]
```diff
@@ -219,7 +219,7 @@ <h3><a>- Barack Obama</a></h3>
                 <div class="col-sm-12">
                     <div class="copyright-text text-center">
                         <p>&copy; 2021; All Rights Reserved.</p>
-                        <p>Designed by <a target="_blank" href="http://ac-uok.com/">Adventure Club</a></p>
+                        <p>Designed by <a target="_blank" href="http://vinojan1999.github.io/">Vinojan Abhimanyu</a></p>
```
Comment (pankajan05) [inline contact.html]: This can't be a personal link. Currently, you are working under the adventure club. You can't link to your personal repo

**Comment by imvinojanv** [comment]
I apologize for some mistakes which I made. 
Thank You

File: game.html [hunk 2]
```diff
@@ -496,8 +496,7 @@ <h2>Send a message</h2>
             <div class="col-sm-12">
                 <div class="copyright-text text-center">
                     <p>&copy; 2021; All Rights Reserved.</p>
-                    <p>Designed by <a target="_blank" href="http://ac-uok.com/">Adventure Club</a></p>
-                </div>
+                    <p>Designed by <a target="_blank" href="http://vinojan1999.github.io/">Vinojan Abhimanyu
```
Comment (pankajan05) [inline game.html]: Need to remove

**Review by pankajan05 — COMMENTED** [review]
Checked

**Pull request merged**